exec/java-exec/src/main/java/org/apache/drill/exec/planner/cost/DrillRelMdDistinctRowCount.java

```diff
@@ -92,7 +92,7 @@ public Double getDistinctRowCount(RelNode rel, RelMetadataQuery mq, ImmutableBit
         if (rel instanceof DrillScanRel) {
           // The existing Drill behavior is to only use this estimation for DrillScanRel and not ScanPrel.
           // TODO: We may potentially do it for ScanPrel (outside the scope of statistics)
-          return rel.estimateRowCount(mq) * 0.1;
+          return ((DrillScanRel)rel).estimateRowCount(mq) * 0.1;
```

Comment:
This cast is not needed, please revert this change.

Reply:
Done.